scripts/build-sunshine.sh

```diff
@@ -4,7 +4,8 @@ set -e
 usage() {
 	echo "Usage: $0"
 	echo "	-d: Generate a debug build"
-	echo "	-p: Generate a debian package"
+	echo "	-p: Generate a linux package"
+	echo "  -e: Extension of package... i.e. 'deb', 'rpm' --> default [deb]"
```

Comment:
Fix indent before `-e` ... sorry, my IDE was screwy here. Actually, don't know why the script is using tabs.

Reply:
Are you using VSCode? VSCode doesn't seem to tell you if it's spaces or tabs unless you select the text.